src/workers/types.ts

```diff
@@ -0,0 +1,65 @@
+import { ImageInfo } from "../Volume";
+import { CreateLoaderOptions } from "../loaders";
+import { LoadSpec, VolumeDims } from "../loaders/IVolumeLoader";
+
+export const enum WorkerMsgType {
+  INIT,
+  CREATE_LOADER,
+  CREATE_VOLUME,
+  LOAD_DIMS,
+  LOAD_VOLUME_DATA,
+}
+
+export const enum WorkerResponseKind {
+  SUCCESS,
+  ERROR,
+  EVENT,
+}
```

Comment:
Nit: Kind vs. Type?

Reply:
Could probably use a more specific word. I wanted to capture that the information this represents (the way in which a request is being responded to) is orthogonal to a message's "type" (the specific action being requested). `WorkerResponseResult`?

Reply:
Result does sound good